6-anthos-install/README.md

```diff
@@ -291,6 +291,31 @@ Create a secret with your private key in both clusters.
 
     kubectl apply --context=${CTX_2} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/acm-configs/config-management-west.yaml
 
+### Install Nomos
+Nomos will be installed to know if the Config Management System changes were successfully applied.
+1. Download the nomos in same version of the Config Management System.
+    ```
+    gsutil cp gs://config-management-release/released/1.7.0/linux_amd64/nomos nomos
+    ```
+
+1. Make it executable:
+    ```
+    chmod +x nomos
+    ```
+
+1. Move the command to a location the system searches for binaries.
+    ```
+    sudo mv nomos /usr/local/bin/
+
+1. Validate the configuration.
+    ```
+    nomos --path=${HOME}/bank-of-anthos-repos/root-config-repo vet
```

Comment:
This path may not always be `${HOME}/bank-of-anthos-repos` right? We should just tell the users to run relative to this repo root.

Reply:
I realized this was in the bastion and we do ask explicitly to use `${HOME}/bank-of-anthos-repos`.